pages/map.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import Head from 'next/head';
+import styled from 'styled-components';
+
+class Map extends React.Component {
+  render() {
+    return (
+      <React.Fragment>
+        <Head>
+          <title>Texas Justice Initiative | Map</title>
+        </Head>
+        <MapIframe src="http://tji-map-viewer.s3-website.us-east-2.amazonaws.com/#4.77/31.67/-100.43" />
+      </React.Fragment>
+    );
+  }
+}
+
+export default Map;
+
+const MapIframe = styled.iframe`
+  height: calc(100vh - 153.375px);
+  width: 100vw;
+  border: 0;
+
+  @media (max-width: ${props => props.theme.medium}) {
+    height: calc(100vh - 90px);
```

Comment:
We define the height of the header as 90px, but maybe this should be pulled out into a variable that we can use here.

https://github.com/texas-justice-initiative/website-nextjs/blob/66bc4c897306a5fca58fce4e674d879274a93843/components/Header.js#L134-L135

Reply:
That would be great. Maybe just a `constants.js` file with an export like `HEADER_HEIGHT` , and we can add to it over time.